### ISSUE TITLE
Ability to view the request like in Chrome

### ISSUE THREAD

**Issue by nbouvrette** [rangav/thunder-client-support#809]
**Is your feature request related to a problem? Please describe.**
Unless I missed something, there is no ability to view the request that was sent which can be useful to troubleshoot issues.

**Describe the solution you'd like**
Like in the Chrome network tab, you can view both the request and response.

**Describe alternatives you've considered**
Postman

**Comment by rangav** [comment]
Hi @nbouvrette 

open the `Output` window, you will see all the request details sent.

**Comment by nbouvrette** [comment]
Wow ok, I spent 5 minutes looking for it... maybe having it in the main UI would make it easier to find?

**Comment by rangav** [comment]
Its a common place where extensions will output log data, may be this is the first time you used that.

**Comment by nbouvrette** [comment]
Yes I use it for debug logs mostly. I think the request is a core component of the functionality. It might be worth considering to move it somewhere it where you have better formatting too.

**Comment by rangav** [comment]
Your feedback is noted, Will think of any better place for logs.
